src/components-creators-insights/sq-modal-activation/sq-modal-activation.component.tsx

```
@@ -0,0 +1,146 @@
+  return (
+    <SqModal
+      className="modal-creators-insights"
+      header={header}
+      footer={footer}
+      open={open}
+      buttonClose={false}
+      modalSize={width > 991 ? 'md' : 'lg'}
+      forceMobileNoMargin={true}
+    >
+      <div className="box-insights">
+        <p dangerouslySetInnerHTML={{ __html: t('insightsText', { socialNetwork: 'instagram' }) }} />
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)